tests/defaults.yml

```diff
@@ -1,7 +1,36 @@
 parameters:
   crossplane:
+    namespace: syn-crossplane
     charts:
-      crossplane: 1.7.0
+      crossplane: 1.1.1
     images:
       crossplane:
-        tag: v1.7.0
+        registry: docker.io
+        image: crossplane/crossplane
+        tag: v1.1.1
+        pullPolicy: IfNotPresent
```

Comment:
The component defaults have never been updated to use v1.7, the initial `tests/defaults.yml` and `tests/openshift4.yml` were created with v1.7 instead of just using the component defaults which I didn't catch in that PR.